PopAppAPI/ClientApp/src/app/models/company/company.ts

```diff
@@ -0,0 +1,8 @@
+export interface Company {
+         CompanyId: number;
+         CompanyName: string;
+         CompanyCode: string;
+         CompanyAdrees: string;
+         CompanyPhone: string;
+         IsActive:boolean;
```

Comment:
Typescript model's properties should be wrote using camel case, example: `companyId: number;`